modules/tinymce/src/core/test/ts/browser/dom/DomUtilsTest.ts

```diff
@@ -10,6 +11,9 @@ import * as HtmlUtils from '../../module/test/HtmlUtils';
 describe('browser.tinymce.core.dom.DOMUtilsTest', () => {
   const DOM = DOMUtils(document, { keep_values: true, schema: Schema() });
 
+  const getTestElement = () =>
```

Comment:
```suggestion
  const getTestElement = (name: string) =>
      DOM.get(name) as HTMLElement;
```
And replace `DOM.get()` below
I don't have a strong opinion about this suggestion. Thought it would  reduce some code

Reply:
That wouldn't reduce code, it'd add a lot more as there's only a couple cases where `test2` or similar is used, but many where `test` is used, so you've have to pass the `test` id through instead. That's why I didn't do it that way... I guess we could have it default to `test` though (e.g. `name: string = 'test'`).